### PR TITLE
update: misc sketch maintenance

### LINKED ISSUE #3477: Sketch Maintenance Mode: Misc sections
REF - [Airtable link](https://airtable.com/appO4WUF5dDhJkMYx/tblj7m3Ll18YYoKZu/viwzn2B8TcGY4Dxor?blocks=hide)

#### Acceptance Criteria

- [ ] All about carbon
- [ ] What's happening
- [ ] Developing
- [ ] Migrating
- [ ] Guidelines
- [ ] Data-viz
- [ ] Help

### LINKED ISSUE #3478: Sketch Maintenance Mode: Contribution & Guidelines sections
REF - [Airtable link](https://airtable.com/appO4WUF5dDhJkMYx/tblj7m3Ll18YYoKZu/viwzn2B8TcGY4Dxor?blocks=hide)

#### Acceptance Criteria

### LINKED ISSUE #3479: Sketch Maintenance Mode: 2x Grid pages
REF - [Airtable link](https://airtable.com/appO4WUF5dDhJkMYx/tblj7m3Ll18YYoKZu/viwzn2B8TcGY4Dxor?blocks=hide)

### LINKED ISSUE #3480: Sketch Maintenance Mode: Color section
REF - [Airtable link](https://airtable.com/appO4WUF5dDhJkMYx/tblj7m3Ll18YYoKZu/viwzn2B8TcGY4Dxor?blocks=hide)

### PR DESCRIPTION
**Pull request by aubrey-oneal**
Closes #3477 
Closes #3478 
Closes #3479
Closes #3480 

#### Changelog

**Changed**
I've split the changes into groups, each which correlate to an issue I have this sprint. I hope by having them in one PR it cuts down on time for review. I also updated incorrect apostrophes as I found them (from ‘ to ').

Group 2: Misc changes
- /all-about-carbon/what-is-carbon/
- /all-about-carbon/what-is-carbon/
- whats-happening/work-in-progress/
- /developing/dev-resources/
- /migrating/guide/overview/
- /migrating/guide/design/
- /migrating/faq/
- /guidelines/accessibility/overview/
- /guidelines/accessibility/overview/
- /guidelines/accessibility/color
- /guidelines/accessibility/color
- /data-visualization/getting-started/
- /help/faq/

Group 3: Contributing
- /contributing/component/
- /contributing/contribute-icons/
- /contributing/contribute-pictograms/

Group 4: 2x grid
- /guidelines/2x-grid/overview/

Group 5: Color
- /guidelines/color/implementation

**Removed**

- Pictogram and Icon contribute tabs (this is duplicated in the contribution pages. Added reroute links)
- how carbon works (hidden page)

### CODE REVIEW

**Comment by alisonjoseph** [comment]
@aubrey-oneal looks like you have several merge conflicts, let me know if you need help resolving them.

**Review by alisonjoseph — APPROVED** [review]
LGTM! We might want to add redirects for the contribute pages that were removed

`/guidelines/icons/contribute` ->` /contributing/contribute-icons/`
`/guidelines/pictograms/contribute` - > `/contributing/contribute-pictograms/`

**Review by tay1orjones — APPROVED** [review]
+1 on the redirects but otherwise looks good to go

File: src/pages/all-about-carbon/who-uses-carbon.mdx [hunk 1]
```diff
@@ -63,12 +63,11 @@ Here are some ways designers can begin engaging with Carbon.
 
 #### Get the tools
 
-- Visit [Designing > Getting Started](/designing/get-started/) on the Carbon
-  website and install the design kits.
+- Visit the [Getting Started](/designing/get-started/) page on the Carbon
```
Comment (aagonzales) [inline src/pages/all-about-carbon/who-uses-carbon.mdx:66]: I know this was already there but do we need "on the Carbon website" part?

File: src/pages/contributing/contribute-pictograms.mdx [hunk 2]
```diff
@@ -61,26 +61,10 @@ guidelines to ensure visual consistency and proper formatting.
 
 ## Exporting SVGs
 
-### Export SVGs from Sketch.
+### Export SVGs from Adobe Illustrator
```
Comment (aagonzales) [inline src/pages/contributing/contribute-pictograms.mdx:64]: Do we need the H3 heading now that there is only one sub-section?

File: src/pages/guidelines/color/implementation.mdx [hunk 3]
```diff
@@ -552,10 +550,11 @@ Carbon color tokens (it would also be to your advantage to use the color token
 layer styles from the Carbon design asset libraries when designing).
 
 Redlining a design in one theme should be enough and not require duplicate
-designs. However, if products teams do want to have design comps in both light
-and dark mode then use this handy Sketch
-[theme switcher](https://github.com/IanKitchen-IBM/carbon-theme-switcher)
-plug-in made by an IBMer.
+designs. However, products teams may want to have design comps in multiple
+themes. In Figma, we recommend that you duplicate your screens, then swap your
```
Comment (aagonzales) [inline src/pages/guidelines/color/implementation.mdx:554]: I believe you can only theme swap the whole file and not specific frames but we may want to check on that. 

```suggestion
themes. In Figma, we recommend that you duplicate your files, then swap your
```

File: src/pages/migrating/faq.mdx [hunk 4]
```diff
@@ -107,27 +107,13 @@ description:
...
-  you are using the Carbon layer styles then the update will be automatic. If
-  you’re using hard values and just specing the token names, then you will need
-  to update your specs.
-
-- Read the
-  [Sketch documentation](https://www.sketch.com/docs/libraries/#updating-library-components)
-  for more details.
+The v10 Carbon libraries are separate from the new v11 Carbon libraries. We
+created separate Figma libraries for v10 and v11 based on user feedback for
+migration purposes. The v10 libraries will not have a continuous update to
```
Comment (aagonzales) [inline src/pages/migrating/faq.mdx:114]: We did this more maintainence and version purposes, not migration. We did it so people didn't have to migrate before they were ready to.
```suggestion
versioning purposes. The v10 libraries will not have a continuous update to
```

Comment (aagonzales) [reply src/pages/migrating/faq.mdx:114]: Or maybe its this ... i like this phrasing in another section
```suggestion
migration preferences. The v10 libraries will not have a continuous update to
```

**Review by aagonzales — COMMENTED** [review]
Looks good! Just a few suggestions and comments

File: src/pages/guidelines/color/implementation.mdx (hunk 3 above)
Comment (aubrey-oneal) [reply src/pages/guidelines/color/implementation.mdx:554]: Thanks! I drafted this and wasn't 100% sure. That's probably the safer instruction.

**Review by aagonzales — APPROVED** [review]

File: src/pages/all-about-carbon/who-uses-carbon.mdx (hunk 1 above)
Comment (laurenmrice) [inline src/pages/all-about-carbon/who-uses-carbon.mdx:66]:
```suggestion
- Visit the [Get started](/designing/get-started/) page on the Carbon
```

Just a small edit to match the name of the actual page it is linked to for consistency.

File: src/pages/all-about-carbon/who-uses-carbon.mdx [hunk 5]
```diff
@@ -63,12 +63,11 @@ Here are some ways designers can begin engaging with Carbon.
...
+  website and install the Figma design kits.
 
 - Visit the [Other Resources](/designing/design-resources/) page on the Carbon
```
Comment (laurenmrice) [inline src/pages/all-about-carbon/who-uses-carbon.mdx:69]:
```suggestion
- Visit the [Other resources](/designing/design-resources/) page on the Carbon
```

File: src/pages/all-about-carbon/who-uses-carbon.mdx [hunk 6]
```diff
@@ -140,8 +139,8 @@ Here are some ways developers can begin engaging with Carbon.
 
 #### Get the tools
 
-- Visit [Developing > Getting Started](/developing/get-started/) on the Carbon
-  website and install your framework of choice.
+- Visit [Developing > Getting Started](/developing/get-started/) and install
```
Comment (laurenmrice) [inline src/pages/all-about-carbon/who-uses-carbon.mdx:142]:
```suggestion
- Visit [Developing > Get started](/developing/get-started/) and install
```

File: src/pages/help/faq/index.mdx [hunk 7]
```diff
@@ -115,6 +115,13 @@ community support for vanilla JS, Angular, Vue, and Svelte implementations.
 Check out [Frameworks](/developing/frameworks/react) to learn more about what
 you need to get started with Carbon implementations.
 
+### Where do I find mobile components?
+
+We are currently working on a Carbon mobile Figma kit. When it's ready, you'll
+find it on the
+[Other Resources](https://carbondesignsystem.com/designing/design-resources/#native-mobile)
```
Comment (laurenmrice) [inline src/pages/help/faq/index.mdx:122]:
```suggestion
[Other resources](https://carbondesignsystem.com/designing/design-resources/#native-mobile)
```

File: src/pages/migrating/guide/design.mdx [hunk 8]
```diff
@@ -42,11 +42,11 @@ accessibility, collaboration, and efficiency for users.
 
 ### Kit migration
```
Comment (laurenmrice) [inline src/pages/migrating/guide/design.mdx:43]: I think this section looks good. However, I wonder if we should cross-reference this part of the page as a link somewhere in the paragraph on the Designing > Get started > [Step 2: Access the tools](https://carbondesignsystem-git-fork-aubrey-142608-carbon-design-system.vercel.app/designing/get-started/#step-2:-access-the-tools) section? So people can find the table and know where to look to get access to the other tools in the meantime if needed. Let me know what you think about that.

**Review by laurenmrice — CHANGES_REQUESTED** [review]
Looks great, Aubrey! ⭐️ Just a couple of small suggestions.

**Review by laurenmrice — APPROVED** [review]

**Review by kingtraceyj — APPROVED** [review]
good to go!

File: src/pages/migrating/guide/design.mdx (hunk 8 above)
Comment (aubrey-oneal) [reply src/pages/migrating/guide/design.mdx:43]: Cool! We discussed in design crit and will put a link under designing/get-started/#additional-design-assets

**Pull request merged**